PyInstaller/loader/rthooks/pyi_rth_osgeo.py

```diff
@@ -0,0 +1,22 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+import os
+import sys
+
+# Installing `osgeo` Conda packages requires to set `GDAL_DATA`
+
+is_win = sys.platform.startswith('win')
+if is_win:
+    gdal_data = os.path.join(sys._MEIPASS, 'Library', 'data')
```

Comment:
Why is an additional directory level used here?
Why not using a more expressive name for the directory, Like `osgeo`